tests/load/pipeline/test_drop.py

```diff
@@ -25,7 +25,17 @@
 
 
 def _attach(pipeline: Pipeline) -> Pipeline:
-    return dlt.attach(pipeline.pipeline_name, pipelines_dir=pipeline.pipelines_dir)
+    p = dlt.attach(
```

Comment:
I'm not sure any test should be modified until we write the test for the feature itself. there may be a few cases where we remove dev_mode during the test. then we'll see